cypress/integration/myPatches/my_patches.ts

```diff
@@ -34,7 +34,9 @@ describe("My Patches Page", () => {
   it("Typing in patch description input updates the url, requests patches and renders patches", () => {
     cy.visit(MY_PATCHES_ROUTE);
     const inputVal = "testtest";
-    cy.dataCy("patch-description-input").type(inputVal);
+    cy.dataCy("patch-description-input").within(() => {
+      cy.get("input").type(inputVal);
```

Comment:
That's an annoying side effect. :/  Is this intentional on leafygreens end?

Reply:
I think so in order to group all the related elements of the form together, including any dropdown options being used. It's too bad it's included when none are used, though.